### ISSUE TITLE
Accessibility: Focus indicator missing [Product Page]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#51923]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

There is no visible focus outline on this element when it receives focus.

![Image](https://github.com/user-attachments/assets/92ad375f-1757-4905-aa6c-d8dfde9dcbf3)

![Image](https://github.com/user-attachments/assets/75cc9b40-891e-49b1-b8b9-a7932fb5190e)

### Component or Page: 

Page: Product Single (Simple)  

### Name affected element:

You may also like section featured image products links

### Relevant code snippets:

```

<a href=""https://woo.equalizedigital.com/product/mediocre-granite-computer/"" class=""woocommerce-LoopProduct-link woocommerce-loop-product__link""><img width=""300"" height=""300"" src=""https://woo.equalizedigital.com/wp-content/uploads/2024/08/img-laudantium-300x300.png"" class=""attachment-woocommerce_thumbnail size-woocommerce_thumbnail"" alt="""" decoding=""async"" loading=""lazy"" srcset=""https://woo.equalizedigital.com/wp-content/uploads/2024/08/img-laudantium-300x300.png 300w, https://woo.equalizedigital.com/wp-content/uploads/2024/08/img-laudantium-100x100.png 100w, https://woo.equalizedigital.com/wp-content/uploads/2024/08/img-laudantium-600x600.png 600w, https://woo.equalizedigital.com/wp-content/uploads/2024/08/img-laudantium-150x150.png 150w, https://woo.equalizedigital.com/wp-content/uploads/2024/08/img-laudantium.png 700w"" sizes=""(max-width: 300px) 100vw, 300px""><h2 class=""woocommerce-loop-product__title"">Group Product Test</h2>
	<span class=""price""><span class=""woocommerce-Price-amount amount""><bdi><span class=""woocommerce-Price-currencySymbol"">$</span>22.00</bdi></span> – <span class=""woocommerce-Price-amount amount""><bdi><span class=""woocommerce-Price-currencySymbol"">$</span>855.57</bdi></span></span>
</a>
```

### Recommended fix:

Modify the CSS in your stylesheet to ensure the element has a 2px solid outline with a 2px outline offset on focus.
